deno/gateway/v10.ts

```diff
@@ -672,6 +678,37 @@ export interface GatewayChannelPinsUpdateDispatchData {
 	last_pin_timestamp?: string | null;
 }
 
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#new-entitlement
+ * https://discord.com/developers/docs/monetization/entitlements#updated-entitlement
+ */
+export type GatewayEntitlementUpdateDispatchData = APIEntitlement;
+
+/**
+ * https://discord.com/developers/docs/monetization/entitlements#new-entitlement
+ * https://discord.com/developers/docs/monetization/entitlements#updated-entitlement
+ */
+export type GatewayEntitlementModifyDispatch = DataPayload<
```

Comment:
Please also export aliases for EntitlementCreate or EntitlementUpdate, pointing to this